clone-master-clean-up.sh

```diff
@@ -15,23 +16,46 @@ err_exit() {
 }
 trap 'err_exit $LINENO' ERR
 
+while [[ $# -gt 0 ]]; do
+    case $1 in
+        -n|--dont-ask)
+            DONT_ASK="YES"
+            shift
+            ;;
+        -f|--dont-change-fstab)
+            DONT_CHANGE_FSTAB="YES"
+            shift
+            ;;
+        -h|--help)
+            echo -e 'Usage: clone-master-clean-up.sh [OPTION..]  \n' \
+                    '  -n, --dont-ask           Do not ask any questions. Run the program with default values even it is not recommended.\n' \
+                    '                           The root password will also be retained.\n' \
+                    '  -f, --dont-change-fstab  Do not swap UUID and label into device name in fstab.\n'
+            exit
+    esac
+done
+
 [ "$UID" != "0" ] && echo 'Please run this program as root user.' && exit 1
 
-echo -e 'The script will delete root SSH keys, log data, and more.\n' \
-     'WARNING: This should only be used on a pristine system\n' \
-     'WARNING: with no populated /home directories!\n' \
-     'Type YES and enter to proceed.'
-read -r answer
-[ "$answer" != "YES" ] && exit 1
-
-if [ -n "$(echo /home/*/.ssh/* /home/*/.*_history)" ]; then
-    echo -e 'There seem to be populated /home directories on this system\n' \
-         'Cloning such systems is not recommended.\n' \
-         'Type YES if you still would like to proceed.'
-    read answer
+if [ "${DONT_ASK}" != "YES" ]; then
+    echo -e 'The script will delete root SSH keys, log data, and more.\n' \
+         'WARNING: This should only be used on a pristine system\n' \
+         'WARNING: with no populated /home directories!\n' \
+         'Type YES and enter to proceed.'
+    read -r answer
     [ "$answer" != "YES" ] && exit 1
 fi
 
+if [ "${DONT_ASK}" != "YES" ]; then
```

Comment:
why 2 if-statements? Can we simplify the code by using only one if-statement?

Reply:
Was wäre ich ohne dich :-)